src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+  width: auto;
```

Comment:
if you change the size of an individual star you would also need to change the size here. if you hardcode some values, do it only in one place. for example, if you have a parent and a child elements, then only set the size for the child. no need to do it for both